### PR TITLE
add nerd bytes page

### PR DESCRIPTION
**Pull request by jaesius**
## Description

Add a Nerd bytes page

### CODE REVIEW

File: src/pages/nerd-bytes.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+                  className={styles.externalLink}
+                  to="https://www.youtube.com/playlist?list=PLmhYj7Jl81JEV-llIDkCVC05tD7fbOv_b"
+                >
+                  Nerd Bytes is a video podcast series
+                  <FeatherIcon
+                    className={styles.externalLinkIcon}
+                    name="external-link"
+                  />
+                </Link>{' '}
+                produced by engineers and practitioners for engineers and
```
Comment (jsbnr) [inline src/pages/nerd-bytes.js:40]:
```suggestion
                produced by engineers and practitioners _for_ engineers and
```

Comment (jsbnr) [reply src/pages/nerd-bytes.js:40]: Italicise the "for"

File: src/pages/nerd-bytes.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                practitioners.
+                <br />
+                <br />
+                Nerd Bytes episodes are crowd sourced, super short, micro topic
+                focused video screencast demos of tips, tricks, features,
+                functionality, news and ideas. Less formal than documentation,
+                easier to digest than a tutorial. Released regularly to
+                continually feed our practitioners with great New Relic content.
```
Comment (jsbnr) [inline src/pages/nerd-bytes.js:48]:
```suggestion
                keep New Relic practitioners up to date with fresh and interesting content.
```

File: src/pages/nerd-bytes.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+              </p>
+              <p>
+                <Button
+                  as={Link}
+                  variant={Button.VARIANT.PRIMARY}
+                  to="https://www.youtube.com/playlist?list=PLmhYj7Jl81JEV-llIDkCVC05tD7fbOv_b"
+                >
+                  Nerd bytes playlist
```
Comment (jsbnr) [inline src/pages/nerd-bytes.js:56]:
```suggestion
                  Nerd Bytes playlist
```

File: src/pages/nerd-bytes.js [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+              className={styles.externalLink}
+              to="https://www.youtube.com/playlist?list=PLmhYj7Jl81JEV-llIDkCVC05tD7fbOv_b"
+            >
+              Nerd Bytes Playlist
+              <FeatherIcon
+                className={styles.externalLinkIcon}
+                name="external-link"
+              />
+            </Link>
+          </p>
```
Comment (jsbnr) [inline src/pages/nerd-bytes.js:83]: Can we include the instructions for subscribing in Slack and MS Teams here?

**Review by jsbnr — CHANGES_REQUESTED** [review]
Just a few changes. Please can we include the instructions on how to subscribe to the You Tube RSS feed of the playlist too.

File: src/pages/nerd-bytes.js [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+                  className={styles.externalLink}
+                  to="https://www.youtube.com/playlist?list=PLmhYj7Jl81JEV-llIDkCVC05tD7fbOv_b"
+                >
+                  Nerd Bytes is a video podcast series
+                  <FeatherIcon
+                    className={styles.externalLinkIcon}
+                    name="external-link"
+                  />
+                </Link>{' '}
+                produced by engineers and practitioners _for_ engineers and
```
Comment (jsbnr) [inline src/pages/nerd-bytes.js:40]: Just checking this appears as italicised yes? no need for <em> tag? Not sure how to see the preview

**Review by jsbnr — APPROVED** [review]
LGTM assuming that _for_ is rendering as expected?

File: src/pages/nerd-bytes.js [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+            >
+              <Video id="sN5GpX0eWMc" type="youtube" />
+            </div>
+          </section>
+
+          <section>
+            <h2>How to consume Nerd Bytes</h2>
+            <p>
+              Find the latest Nerd Bytes episode on the New Relic YouTube
+              channel{' '}
```
Comment (jpvajda) [inline src/pages/nerd-bytes.js:76]: is this `{'  '}` required here?

File: src/pages/nerd-bytes.js [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+                className={styles.externalLink}
+                to="https://www.youtube.com/playlist?list=PLmhYj7Jl81JEV-llIDkCVC05tD7fbOv_b"
+              >
+                Nerd Bytes Playlist
+                <FeatherIcon
+                  className={styles.externalLinkIcon}
+                  name="external-link"
+                />
+              </Link>
+              . But, we encourage you to subscribe to the Nerd Bytes RSS feed
```
Comment (jpvajda) [inline src/pages/nerd-bytes.js:87]:
```suggestion
              We encourage you to subscribe to the Nerd Bytes RSS feed
```

File: src/pages/nerd-bytes.js [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+              and add it to your Slack or Team channel. Fresh content will be
+              delivered directly to you when it is published.{' '}
```
Comment (jpvajda) [inline src/pages/nerd-bytes.js:89]: is this {' '} required here?

File: src/pages/nerd-bytes.js [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+          <section
+            className={cx(
+              styles.section,
+              styles.stripedSection,
+              styles.developerChampions
+            )}
+          >
+            <div>
+              <h2>
+                Want all the news, code and pontification that's fit for
```
Comment (jpvajda) [inline src/pages/nerd-bytes.js:126]:
```suggestion
                Do you want all the news, code and pontification that's fit for
```

**Review by jpvajda — CHANGES_REQUESTED** [review]
Had a few suggestions...

File: src/pages/nerd-bytes.js (hunk 9 above)
Comment (jaesius) [reply src/pages/nerd-bytes.js]: This change has been added

File: src/pages/nerd-bytes.js (hunk 8 above)
Comment (jaesius) [reply src/pages/nerd-bytes.js]: Space wasn't needed and has been removed.

File: src/pages/nerd-bytes.js (hunk 7 above)
Comment (jaesius) [reply src/pages/nerd-bytes.js]: Updated

File: src/pages/nerd-bytes.js (hunk 6 above)
Comment (jaesius) [reply src/pages/nerd-bytes.js:76]: This was required for spacing

**Comment by jpvajda** [comment]
@jaesius there looks to be some merge coflicts in this PR now...

**Review by jpvajda — CHANGES_REQUESTED** [review]
please resolve merge conflicts

**Review by jpvajda — COMMENTED** [review]
I'm a little concerned with this commit as there are a lot of changes to gatsby-config.js which is sort of odd..I'm going to ask the DevEn hero to provide some guidance here.

**Comment by roadlittledawn** [comment]
> I'm a little concerned with this commit as there are a lot of changes to gatsby-config.js which is sort of odd..I'm going to ask the DevEn hero to provide some guidance here.

@jpvajda . i'm not seeing changes to `gatsby-config.js` in this PR. what in particular should i be looking for? at quick glance, changes look okay. will pull down and check that page just to make sure

**Review by roadlittledawn — APPROVED** [review]
This looks okay to me. Didn't see any changes to `gatsby-config.js` and page renders fine locally

**Review by jpvajda — APPROVED** [review]

**Pull request merged**